apache-solr/scripts/load_generation.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
A bit of a thought exercise, with no real call to action.

Would this be better/worse/different if we used k6 to run these tests?

Reply:
cool, I'll play with this more in detail. We have a free day coming up, so I can spend some time trying this out for future sample apps.